bgpd/bgp_vty.c

```diff
@@ -8112,22 +8117,25 @@ static void bgp_show_summary_afi_safi(struct vty *vty, struct bgp *bgp, int afi,
 
 	if (use_json && is_wildcard)
 		vty_out(vty, "}\n");
-	else if (use_json && !json_output)
-		vty_out(vty, "{}\n");
+	else if (!nbr_output)
+		use_json ? vty_out(vty, "{}\n")
+			 : vty_out(vty, "%% No BGP neighbors found\n");
```

Comment:
If I understand what you're objecting to, I could have used the hook operator just on the string supplied to vty_out instead of using for vty_out itself.  Is that correct?   I agree that may be better and can change it to that.

Reply:
After changing, I agree it's more concise and easier to read.  Will push changes for all of them shortly